specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/assessmentMetadata.json

```diff
@@ -193,7 +193,7 @@
           "Assessments Metadata"
         ],
         "description": "Create metadata information on an assessment type in a specific subscription",
-        "operationId": "AssessmentsMetadataSubscription_Create",
+        "operationId": "AssessmentsMetadata_Create",
```

Comment:
Why this is not `CreateInSubscription` to align the other two method names?

Reply:
```suggestion
        "operationId": "AssessmentsMetadata_CreateInSubscription",
```